mod-infra/src/main/scala/at/forsyte/apalache/infra/log/LogbackConfigurator.scala

```diff
@@ -37,7 +40,8 @@ class LogbackConfigurator(runDirOpt: Option[Path]) extends ContextAwareBase with
     val consoleAppender = mkConsoleAppender(loggerContext)
     // only warnings at the root level
     rootLogger.setLevel(Level.WARN)
-    rootLogger.addAppender(mkFileAppender(loggerContext))
+    runDir.foreach(d => rootLogger.addAppender(mkFileAppender(loggerContext, d.resolve("detailed.log").toFile())))
+    customRunDir.foreach(d => rootLogger.addAppender(mkFileAppender(loggerContext, d.resolve("detailed.log").toFile())))
```

Comment:
You can simplify this with option magic:
```suggestion
    (runDir ++ customRunDir).foreach(d => rootLogger.addAppender(mkFileAppender(loggerContext, d.resolve("detailed.log").toFile())))
```

Reply:
That is cleaner. Thanks!